Source/AsyncDisplayKit-Prefix.pch

```diff
@@ -8,23 +8,4 @@
   #import <Foundation/Foundation.h>
 #endif
 
-// Some build systems (Cocoapods, Buck, Bazel, etc) will define these flags manually if the functionality
-// is needed by the app. Carthage in particular, or if a user forgets to set the build flag, benefit from
-// checking if each flag is not defined and then setting it to whether or not the header is accessible.
-
-#ifndef PIN_REMOTE_IMAGE
-// For Carthage or manual builds, this will define PIN_REMOTE_IMAGE if the header is available in the
-// search path e.g. they've dragged in the framework (technically this will not be able to detect if
-// a user does not include the framework in the link binary with build step).
-  #define PIN_REMOTE_IMAGE __has_include(<PINRemoteImage/PINRemoteImage.h>)
-#endif
-
-#ifndef IG_LIST_KIT
-  #define IG_LIST_KIT __has_include(<IGListKit/IGListKit.h>)
-
-  /**
-   * For IGListKit versions < 3.0, you have to use IGListCollectionView.
-   * For 3.0 and later, that class is removed and you use UICollectionView.
-   */
-  #define IG_LIST_COLLECTION_VIEW __has_include(<IGListKit/IGListCollectionView.h>)
-#endif
+#import <AsyncDisplayKit/ASAvailability.h>
```

Comment:
I thought the idea / main benefit of this was to not rely on the pch...indeed, there are a number of build systems where getting the pch to work correctly has been a challenge (even with Cocoapods and Carthage, I've seen issues with this in the last couple months like when working on Yoga)

What are your thoughts about keeping the explicit include in other files? If any code attempts to use one of these defines and it hasn't been defined, there should be a compiler error so it won't fail silently.

If you prefer this approach, I will try to make it work — but it's less clear to me what the benefit of moving it to ASAvailability is if the pch is still essential.